apps/web/components/ui/UsernameAvailability/PremiumTextfield.tsx

```diff
@@ -202,6 +195,20 @@ const PremiumTextfield = (props: ICustomUsernameProps) => {
     }
   };
 
+  let paymentMsg = !currentUsername ? (
+    <span className="text-xs text-orange-400">
+      You need to reserve your premium username for {getPremiumPlanPriceValue()}
```

Comment:
Should this be untranslated?

Reply:
Yeah, it makes sense to translate it. Will translate this and other strings in a followup PR, once we have this important PR merged.